h4/h4.ino

```diff
@@ -1666,7 +1666,7 @@ void setup() {
   isOn = false;
   savedDupr = dupr = pref.getLong(PREF_DUPR);
   motionMutex = xSemaphoreCreateMutex();
-  savedStarts = starts = min(STARTS_MAX, max(1, pref.getInt(PREF_STARTS)));
+  savedStarts = starts = min(static_cast<long>(STARTS_MAX), max(static_cast<int32_t>(1), pref.getInt(PREF_STARTS)));
```

Comment:
This design is difficult for me. I could be wrong.

Reply:
Can you just use `(int32_t) 1` style of conversion?

Reply:
So?
https://github.com/kachurovskiy/nanoels/pull/210/commits/8d1285259bee776ec448dcc5d7a11206e350ce1a